zimbra/src/main/java/fr/openent/zimbra/helper/EventBusHelper.java

```diff
@@ -0,0 +1,20 @@
+package fr.openent.zimbra.helper;
+
+import fr.openent.zimbra.controllers.ZimbraController;
+import fr.openent.zimbra.core.constants.Field;
+import io.vertx.core.eventbus.Message;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+public class EventBusHelper {
+
+    private static final Logger log = LoggerFactory.getLogger(ZimbraController.class);
+
+    public static void eventBusError(String errMessage, String errI18n, Message<JsonObject> message) {
```

Comment:
```suggestion
    public static void eventBusError(String logMessage, String replayErrorMessage, Message<JsonObject> message) {
```
Lorsque tu nomme tes variables, il ne faut pas que tu te base sur le contexte de ce qui l'utilise. Ici tu as nommé errI18n car la fonction parent va appeler cette méthode avec un variable qui correspond a un i18n. Mais si j'utilise t'as méthode, la variable que je vais fournir ne sera pas forcément un i18n.